tools/rpkg/tests/testthat/test_register_arrow.R

```diff
@@ -235,34 +225,33 @@ test_that("duckdb_register_arrow() performs selection pushdown time type", {
     duckdb::duckdb_register_arrow(con, "testarrow", arrow_table)
 
     # Try ==
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a ='00:01:00'")[[1]], 1)
-    # Try >
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a >'00:01:00'")[[1]], 2)
-    # Try >=
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a >='00:10:00'")[[1]], 2)
-    # Try <
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a <'00:10:00'")[[1]], 1)
-    # Try <=
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a <='00:10:00'")[[1]], 2)
-
-    # Try Is Null
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a IS NULL")[[1]], 1)
-    # Try Is Not Null
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a IS NOT NULL")[[1]], 3)
-
-    # Try And
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a='00:10:00' and b ='00:01:00'")[[1]], 0)
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a ='01:00:00' and b = '00:10:00' and c = '01:00:00'")[[1]], 1)
-    # Try Or
-    expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a = '01:00:00' or b ='00:01:00'")[[1]], 2)
+    expect_error(dbGetQuery(con, "SELECT count(*) from testarrow where a ='00:01:00'"))
+    # # Try >
+    # expect_equal(dbGetQuery(con, "SELECT count(*) from testarrow where a >'00:01:00'")[[1]], 2)
```

Comment:
Should this be commented out as well then?

Reply:
Yeah, so the time filter pushdown doesn't work on R because of a missing cast on the arrow side. I guess I could also have expect_error on the remaining code.